MapleServer2/PacketHandlers/Game/RequestItemBreakHandler.cs

```diff
@@ -67,9 +68,20 @@ private static void HandleAdd(GameSession session, PacketReader packet)
         private static void HandleRemove(GameSession session, PacketReader packet)
         {
             long uid = packet.ReadLong();
-            short key = session.Player.DismantleSlots.FirstOrDefault(x => x.Value == uid).Key;
+            Tuple<long, int>[] dismantleSlots = session.Player.DismantleSlots;
+            for (int i = 0; i < dismantleSlots.Length; i++)
```

Comment:
You can probably just find the index having a null slot using Linq

Reply:
Not sure what you mean by null slot.

Reply:
I tried but array doesn't have Find?